tests/NLog.UnitTests/Targets/AsyncTaskTargetTest.cs

```diff
@@ -357,6 +357,31 @@ public void AsyncTaskTarget_TestSlowBatchWriting()
 
             Assert.True(DateTime.UtcNow - utcNow < TimeSpan.FromSeconds(4));
         }
+
+        [Fact]
+        public void AsyncTaskTarget_TestThrottleOnTaskDelay()
+        {
+            ILogger logger = LogManager.GetCurrentClassLogger();
+
+            var asyncTarget = new AsyncTaskBatchTestTarget
+            {
+                Layout = "${level}",
+                TaskDelayMilliseconds = 50,
+                BatchSize = 10,
+            };
+
+            SimpleConfigurator.ConfigureForTargetLogging(asyncTarget, LogLevel.Trace);
+            Assert.True(asyncTarget.Logs.Count == 0);
```

Comment:
Is this assert really needed? I really prefer AAA ;)

Reply:
Fixed